internal/auth/oidc/auth_method.go

```diff
@@ -106,6 +106,10 @@ func NewAuthMethod(scopeId string, clientId string, clientSecret ClientSecret, o
 			a.SigningAlgs = append(a.SigningAlgs, string(alg))
 		}
 	}
+	if len(opts.withClaimsScopes) > 0 {
+		a.ClaimsScopes = make([]string, 0, len(opts.withClaimsScopes))
```

Comment:
I realize it gets round-tripped back through the auth method but I guess I'd assume that what went into the options is unlikely to be used further...do you think it's safe to avoid this copy?

Reply:
I agree, that it's not needed.  ty!